v0/destinations/awin/config.js

```diff
@@ -0,0 +1,8 @@
+const BASE_URL = "https://www.awin1.com/sread.php?tt=ss&tv=2";
```

Comment:
```suggestion
const BASE_URL = "https://www.awin1.com/sread.php";
```
We shouldn't provide the query params here.

Reply:
Removed